src/components/Tabs.js

```diff
@@ -205,4 +128,184 @@ class TabWithoutRadium extends React.Component {
 
 const Tab = Radium(TabWithoutRadium);
 
+class MenuTabs extends StandardTabs {
```

Comment:
@lukeschunk I tend to favor composition over inheritance, but I felt that this was an excellent use case for inheritance. These polymorphic tab components are not exported and are very similar in behavior so I thought it made sense.

How would you go about refactoring this to use composition? Do you want to experiment with it and throw up a PR against this branch?